pool/web/handler/admin_experiments_assignments.ml

```diff
@@ -158,31 +158,28 @@ module Access : sig
   val cancel : Rock.Middleware.t
   val mark_as_deleted : Rock.Middleware.t
 end = struct
+  open Guard
   module AssignmentCommand = Cqrs_command.Assignment_command
+  module Guardian = Middleware.Guardian
 
-  let assignment_effects =
-    Middleware.Guardian.id_effects Assignment.Id.of_string Field.Assignment
-  ;;
+  let read_effect = EffectSet.One (Action.Read, TargetSpec.Entity `Assignment)
```

Comment:
Shouldn't this middleware include an experiment specific rule, too? Like e.g. the mailing handler

Please also check `admin_experiments_invitations.ml` & `admin_experiments_waiting_list.ml`